js/menus/menuTemplate.js

```diff
@@ -156,9 +171,73 @@ function getTemplate(app) {
                 role: 'front'
             }
         ]
+    } else {
+      // Window menu when Windows.
+        template[2].submenu.push(
+            {
+                label: 'Auto Launch on Windows Startup', 
+                type: 'checkbox', 
+                checked: launchOnStartup,
+                click: function (item) {
+                    if (item.checked){
+
+                        symphonyAutoLauncher.isEnabled()
+                        .then(function(isEnabled){
+                            if(isEnabled){
+                                return;
+                            }
+                            symphonyAutoLauncher.enable();
+                        })
+                        .catch(function(err){
+                            let title = 'Error setting Symphony Auto Launch';
+                            electron.dialog.showErrorBox(title, title + ': ' + err);
+                        });
+
+                    } else {
+                        symphonyAutoLauncher.disable();
+                    }
+                    launchOnStartup = item.checked;
+                    updateConfigField('launchOnStartup', launchOnStartup);
+                }
+            }
+        )
     }
 
+    // Window menu -> minimizeOnClose.
+    // ToDo: Add behavior on Close.
+    var index = 2;
+    if (process.platform === 'darwin' && template[0].label !== app.getName()){
+        index = 3;
+    }
+    template[index].submenu.push(
```

Comment:
is the minimize feature on close completed?  mac currently has the feature by default, see windowMgr.js.   this flag should be used there to enable/disable close.